db/migrate/20141214220050_create_proposals.rb

```diff
@@ -0,0 +1,37 @@
+class CreateProposals < ActiveRecord::Migration
+  def change
+    create_table :proposals do |t|
+      t.string :status
+      t.string :flow
+      t.timestamps
+    end
+    add_reference :carts, :proposal
```

Comment:
Will NCR be of type Cart? Would it be worthwhile to name that appropriately at this point? I'm also happy with capturing that in a separate story.